modules/rhcos-install-iscsi-ibft.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+// * installing/installing_bare_metal/installing-bare-metal-network-customizations.adoc
+// * installing/installing_bare_metal/installing-restricted-networks-bare-metal.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="rhcos-install-iscsi-ibft_{context}"]
+= Installing {op-system} on an iSCSI boot device using iBFT
+
+On a completely diskless machine, the iSCSI target and initiator values can be passed through iBFT. This procedure supplies these values using an iPXE boot script and uses a multipath iSCSI target.
+
+.Prerequisites
+. You are in the {op-system} live environment.
+. You have enabled multipathing on your primary disk.
+
+.Procedure
+
+. Create an iPXE boot script that contains the iSCSI target and initiator values:
++
+[source,bash]
+----
+#!ipxe
+set initiator-iqn iqn.<UUID>
+sanboot iscsi:127.0.0.1::::iqn.2024-05.com.coreos:0 \
+        iscsi:127.0.0.1::::iqn.2024-05.com.coreos:1 <1>
+----
+<1> A multipath iSCSI target. See "Enabling multipathing with kernel arguments on {op-system}" for more information on multipathing.
+
+. Mount the iSCSI target from the live environment:
++
+[source,text]
+----
+$ iscsiadm \
+    --mode discovery \
+    --type sendtargets
+    --portal 127.0.0.1 \
+    --login
+----
+
+. Install {op-system} onto the iSCSI target using the necessary kernel arguments, specifying `ip=ibft`:
++
+[source,text]
+----
+$ coreos-installer install \
+    /dev/disk/by-path/ip-127.0.0.1\:3260-iscsi-iqn.2024-05.com.coreos\:0-lun-0 \
+    --append-karg rd.iscsi.firmware=1 \ <1>
+    --append-karg ip=ibft \ <2>
```

Comment:
I would default to leaving this off. We already bring up all interfaces during the initrd boot, which will include the interface we booted from. `ip=ibft` is only if you want to bring up *just* the interface we booted from, which is a less safe default. I'd leave it undocumented in these docs IMO.